scripts/project-board-summary.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
```

Comment:
We should add a section about this tool to the [Authoring and sending a newsletter](https://github.com/Financial-Times/origami-website/tree/2020-06-01-newsletter#authoring-and-sending-a-newsletter) documenation